### DISCUSSION TITLE
Customer bill, Customer Receipt Lables

### DISCUSSION THREAD

**Discussion by mbugikp** [Blair2004/NexoPOS#1374] [Ideas]
**Describe the bug**
When selling the steps are normally ln the following order,
Captain Order/Customer order,
Customer Bill,
Customer Receipt,

We should clearly label the customer bill (all orders on hold), after payment We should again label the Receipt  

**To Reproduce**
Sell, Print. Check print out.

**Expected behavior**
When presented with a bill Customer should clearly know it is a bill we therefore should add the words "Customer Bill", After Payment the word should change to "Customer Receipt or Just Receipt"

**Screenshots**

**Environment:**
 - NexoPOS 4.x vesrion: [e.g. 4.5.2]
 - Premium Extensions: [e.g Nexo Print Adapter (4.5.0), Gastro (4.5.2)]

**Comment by Blair2004** [comment]
Hi, not sure to understand... "Captain Order/ Customer Order" ???

**Comment by mbugikp** [comment]
Kitchen order

**Comment by mbugikp** [comment]
The order that goes to the kitchen

**Comment by Blair2004** [comment]
Hi, 
Thank you for the clarification. Not sure if that is a urgent needs. In fact, on each receipt we either mention if it's paid or not. Having to change the label for each print, honestly will have us thinking about unnecessary complexity. 

I'll keep it aside for now.

Thank you.

**Reply by mbugikp** [reply]
In my opinion if the payment status is hold it should correctly label that as "Bill", If payment status is not hold then the label changes to "Receipt".
My suggestion.

Regards.

**Comment by mbugikp** [comment]
If the payment status is unpaid=the label changes to Bill, if amount paid is greater than 0 then label changes to Receipt

**Reply by Blair2004** [reply]
What about partially paid order?

**Comment by mbugikp** [comment]
Partial paid order is a receipt

**Reply by mbugikp** [reply]
A receipt with balance

**Reply by Blair2004** [reply]
Alright thank you for your contribution... Really much appreciated

**Reply by mbugikp** [reply]
When a customer pays any amount greater than one or equal to one we should correctly label that as a receipt, even if the receipt is partially paid.
